packages/knowledge/src/base.ts

```diff
@@ -678,17 +678,20 @@ export const base: ChartKnowledgeBase = {
   },
 
   boxplot: {
-    id: 'boxplot',
-    name: '',
-    alias: [],
-    family: [],
-    def: '',
-    purpose: [],
-    coord: [],
-    category: [],
-    shape: [],
-    dataPres: [],
-    channel: [],
+    id: 'box_plot',
+    name: 'Box Plot',
+    alias: ['Box and Whisker Plot'],
```

Comment:
so 'boxplot' could be an alias

```suggestion
    alias: ['Box and Whisker Plot', 'boxplot'],
```